test/internal/pbxproj_partials/write_files_and_groups_tests.bzl

```diff
@@ -185,8 +197,8 @@ write_files_and_groups_test = unittest.make(
         "file_paths": attr.string_list(mandatory = True),
         "files": attr.string_list(mandatory = True),
         "folders": attr.string_list(mandatory = True),
-        "generated_files": attr.string_list(mandatory = True),
-        "generated_folders": attr.string_list(mandatory = True),
+        "generated_files": attr.string_list_dict(mandatory = True),
```

Comment:
Should this instead be `attr.string_dict` so we can express the 1:1 nature? We can't have multiple owners for a file.

Reply:
The ultimately gets transformed into a `struct` to represent a `File` and my intent was to make this extensible in the future to enable mocking other fields (aside from just `owner`) if the implementation expands to use other fields on `File`. wdyt?

Reply:
I don't think we need to future proof it.